src/backend/executor/execUtils.c

```diff
@@ -2191,16 +2192,12 @@ void mppExecutorFinishup(QueryDesc *queryDesc)
 			foreach(lc, estate->es_result_aosegnos)
 			{
 				SegfileMapNode *map = lfirst(lc);
-				struct {
-					Oid relid;
-			   		int64 tupcount;
-				} *entry;
 				bool found;
 
-				entry = hash_search(aopartcounts,
-									&(map->relid),
-									HASH_FIND,
-									&found);
+				PQaoRelTupCount *entry = hash_search(aopartcounts,
```

Comment:
It looks that we do not actually need the delta in execUtils.c to pass the test. If it is an additional refactoring, we should do it via a separate ticket & PR.

Reply:
You’re right, it’s not related to COPY processing. It’s related to INSERT processing, so I will remove it.